tests/test_reporters.py

```diff
@@ -661,3 +661,13 @@ def test_html_report_delta__show_missing_False():
     </div>
   </body>
 </html>"""
+
+def test_github_report():
+    from pycobertura.reporters import GitHubAnnotationReporter
+
+    cobertura = make_cobertura()
+    report = GitHubAnnotationReporter(cobertura)
+
+    assert report.generate() == """\
+::error file=search/BinarySearch.java,line=24,endLine=24,title=pycobertura::not covered
+::error file=search/LinearSearch.java,line=19,endLine=24,title=pycobertura::not covered"""
```

Comment:
@aconrad this is probably the best.
However, I would suggest snapshotting the test outputs with [syrupy](https://github.com/tophat/syrupy) or some similar packages, but it is clearly out of scope

Reply:
Thanks for the suggestions, I didn't know about this library. 👍

As an alternative, we can use pytest's parametrize to avoid repeating tests. Here's an example usage:

https://github.com/aconrad/pycobertura/blob/3c0d01c601dcbd12cfc21ac6609e43fc205b1c7f/tests/test_cobertura.py#L108-L118

And here's the documentation for it:

https://docs.pytest.org/en/6.2.x/parametrize.html

Reply:
https://github.com/aconrad/pycobertura/pull/164/commits/b220a6e508d4c508b2f785f2530c7029ce57a5a5